package.json

```diff
@@ -17,6 +17,7 @@
         "@types/node": "^10.17.27",
         "@typescript-eslint/eslint-plugin": "^4.23.0",
         "@typescript-eslint/parser": "^4.23.0",
+        "@shapirov/cdk-eks-blueprint": "0.12.0",
```

Comment:
This should be removed from this `package.json`

Reply:
resolved.